lib/WP_Auth0_LoginManager.php

```diff
@@ -61,38 +126,43 @@ public function login_auto() {
 			// Create the link to log in.
 			$login_url = 'https://' . $this->a0_options->get( 'domain' ) .
 						'/authorize?' .
-						'scope=' . urlencode( $options['auth']['params']['scope'] ) .
```

Comment:
For others reading: https://stackoverflow.com/questions/996139/urlencode-vs-rawurlencode